CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [7.0.2] - 2024-03-21
```

Comment:
Changes looks good to me, but since this is a new feature we going to support, so I suggest to bump the version to `7.1.0`

Reply:
Updated version to 7.1.0 in CHANGELOG